distribution/docs/src/main/jdocs/content/_sources/wfs-20-source-contents.adoc

```diff
@@ -36,9 +36,7 @@ Configure an existing WFS v2.0.0 Source through the ${admin-console}:
 * Select the *Sources* tab.
 * Select the name of the source to edit.
 
-include::{adoc-include}/_tables/Wfs_v2_0_0_Federated_Source-table-contents.adoc[]
-
-include::{adoc-include}/_tables/Wfs_v2_0_0_Connected_Source-table-contents.adoc[]
+See <<Wfs_v2_0_0_Federated_Source,WFS v.2.0 Federated source>> or <<Wfs_v2_0_0_Connected_Source,WFS v2.0 Connected source>> configurations for all possible configurations.
```

Comment:
See <<Wfs_v2_0_0_Federated_Source,WFS v.2.0 Federated **S**ource **configurations**>> or <<Wfs_v2_0_0_Connected_Source,WFS v2.0 Connected **S**ource **configurations**>> for all possible configurations.